Groups/GroupFinder.ascx.cs

```diff
@@ -78,7 +78,7 @@ namespace RockWeb.Plugins.rocks_kfs.Groups
     [CampusField( "Default Location", "The campus address that should be used as fallback for the search criteria.", false, "", "" )]
     [BooleanField( "Single Select Campus Filter", "When set to true, the campus filter will be a drop down instead of checkbox.", false, key: "SingleSelectFilters" )]
     [BooleanField( "Allow Search in PersonGuid Mode", "When set to true PersonGuid mode will allow you to change filters and search in that mode for that person.", false, key: "AllowSearchPersonGuid" )]
-    [BooleanField( "Collapse Filters on Search", "When set to true, all filters will be collapsed into a single 'Filters' dropdown.", false )]
+    [CustomDropdownListField( "Collapse Filters on Search", "When set to yes, all filters will be collapsed into a single 'Filters' dropdown.", "False^No,True^Yes,InitialLoad^Same as Initial Load", false, "False" )]
```

Comment:
Link Hide Filters on Load setting with this by changing word to Collapse

Reply:
@guymaness2 changes ready for review: [`12dae1d` (#71)](https://github.com/KingdomFirst/RockBlocks/pull/71/commits/12dae1d22a95bba34ad55242c29906e985369336)